templates/connect-inject-clusterrole.yaml

```diff
@@ -33,7 +31,7 @@ rules:
   verbs:
   - use
 {{- end }}
-{{- if and .Values.global.acls.manageSystemACLs (or .Values.global.enableConsulNamespaces .Values.connectInject.healthChecks.enabled) }}
```

Comment:
Previously we'd only need an acl token if namespaces or healthchecks were enabled but now we always need one because cleanup controller is on by default (and no way right now to turn it off through helm)